cmd/admin-subnet-health.go

```diff
@@ -29,10 +33,33 @@ var adminSubnetHealthCmd = cli.Command{
 	Action:             mainSubnetHealth,
 	Before:             setGlobalsFromContext,
 	Hidden:             true,
-	CustomHelpTemplate: "Please use 'mc support diagnostics'",
+	Flags:              append(supportDiagFlags, globalFlags...),
+	CustomHelpTemplate: "Please use 'mc support diag'\n",
```

Comment:
Should we be more specific?

"This command is deprecated and will be removed in a future release. Use `mc support diag` instead."

Reply:
Done